ppml/base/Dockerfile

```diff
@@ -83,7 +83,22 @@ RUN env DEBIAN_FRONTEND=noninteractive apt-get update && \
     apt-get update && \
     apt-get install -y libsgx-enclave-common-dev  libsgx-ae-qe3 libsgx-ae-qve libsgx-urts libsgx-dcap-ql libsgx-dcap-default-qpl libsgx-dcap-quote-verify-dev libsgx-dcap-ql-dev libsgx-dcap-default-qpl-dev libsgx-quote-ex-dev libsgx-uae-service libsgx-ra-network libsgx-ra-uefi
 
+# java
+ARG JDK_VERSION=8u192
+ARG JDK_URL
+ENV JAVA_HOME           /opt/jdk$JDK_VERSION
+ENV PATH                ${JAVA_HOME}/bin:${PATH}
+RUN wget $JDK_URL && \
+    gunzip jdk-$JDK_VERSION-linux-x64.tar.gz && \
+    tar -xf jdk-$JDK_VERSION-linux-x64.tar -C /opt && \
+    rm jdk-$JDK_VERSION-linux-x64.tar && \
+    mv /opt/jdk* /opt/jdk$JDK_VERSION && \
+    ln -s /opt/jdk$JDK_VERSION /opt/jdk
 
+ADD ./verification_jars /ppml/verification_jars
```

Comment:
we'd better download jars in the dockerfile rather than adding them
why do we need spark jars? log4j jar may have critical vulnerabilities, need to check